src/forms/input-group.scss

```diff
@@ -18,6 +18,12 @@
   &.inline {
     display: inline-table;
   }
+
+  // Autocomplete with embedded icon
+  .form-control.autocomplete-embedded-icon-wrap {
+    display: inline-flex;
+    padding: $spacer-1*1.25 $spacer-2;
```

Comment:
Can you also make this change for the other autocomplete form controls? `$spacer-1 * 1.25`

I feel like it needs a space in the math but maybe it doesn't matter 🤔

Reply:
Fixed!